app/db/impl/access/UserBase.scala

```diff
@@ -108,13 +107,13 @@ class UserBase(override val service: ModelService,
   def getStaff(ordering: String = ORDERING_ROLE, page: Int = 1)(implicit ec: ExecutionContext): Future[Seq[User]] = {
     // determine ordering
     val (sort, reverse) = if (ordering.startsWith("-")) (ordering.substring(1), false) else (ordering, true)
-    val staffRoles = List(RoleTypes.Admin, RoleTypes.Mod)
+    val staffRoles: List[RoleType] = List(RoleType.OreAdmin, RoleType.OreMod)
```

Comment:
Any reason for specifying the type here?

Reply:
Yes. I tried it without but it threw a compiler error since slick didn't knew which type to bind on with the `bind` method used below.